src/frontend/apps/desk/src/features/header/AccountDropdown.tsx

```diff
@@ -9,7 +9,7 @@ export const AccountDropdown = () => {
   const { t } = useTranslation();
   const { userData, logout } = useAuthStore();
 
-  const userName = userData?.name || t('No Username');
+  const userName = userData?.name || userData?.email || t('No Username');
```

Comment:
```suggestion
  const userName = userData?.name || userData?.email;
```

`User.email` is not nullable, so the last alternative no longer needs to appear.

Reply:
@Morendil User.email can be null
--> https://github.com/numerique-gouv/people/blob/main/src/backend/core/models.py#L439